pkg/sensors/tracing/generickprobe.go

```diff
@@ -544,6 +598,65 @@ func loadGenericKprobeSensor(bpfDir, mapDir string, load *program.Program, versi
 	return err
 }
 
+func loadMultiKprobeSensor(ids []idtable.EntryID, bpfDir, mapDir string, load *program.Program, version, verbose int) error {
+	sensors.AllPrograms = append(sensors.AllPrograms, load)
+
+	bin_buf := make([]bytes.Buffer, len(ids))
+
+	for index, id := range ids {
+		gk, err := genericKprobeTableGet(id)
+		if err != nil {
+			return err
+		}
+
+		if !load.RetProbe {
+			load.MapLoad = append(load.MapLoad, selectorsMaploads(gk.loadArgs.selectors, gk.pinPathPrefix, uint32(index))...)
+		}
+
+		binary.Write(&bin_buf[index], binary.LittleEndian, gk.loadArgs.config)
+		config := &program.MapLoad{
+			Index: uint32(index),
```

Comment:
Maybe I'm reading the code incorrectly, but what would happen if we have more than one tracing policy with generic kprobes when we use multi-kprobes?

From what I can tell, because the maps are shared across _all_ policies we would break the previously loaded kprobes.  for example, we would override their maps because, AFAICT, there is a single config map for all multi-kprobes.

So I wonder if it makes sense to share maps and programs only across a single sensor. (see comment in PinFile)

Reply:
ah right, I did not realized the sensorPath change got merged.. should have checked the rebase more carefuly :-\
true, it all needs to go under sensorPath, will change, thanks